data/config-schema/src/main/resources/net/corda/schema/cordapp/configuration/external.messaging/1.0/corda.external.messaging.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Possibly, it would be good to keep this file out of this PR such that this PR will only be focused on refactoring for Config and CorDappConfig.

Then, PR to follow will do everything necessary to fit External Messages in.

Reply:
Splitting this PR into two seems to require a lot more effort due to the process of merging changes to the api repository. I would suggest not to split this PR due to time constraints. I hope this is fine with you, @vkolomeyko.

Reply:
Sure, let's leave it as one PR.